src/main/services/pre-api/pre-client.ts

```diff
@@ -25,7 +31,55 @@ export class PreClient {
 
       return response.data as Recording;
     } catch (e) {
-      return null;
+      if (e.response.status === 404) {
+        return null;
+      }
+
+      this.logger.error(e);
+      throw e;
+    }
+  }
+
+  public async getRecordingPlaybackData(id: string): Promise<RecordingPlaybackData | null> {
+    const url = config.get('ams.flowUrl') as string;
+    const key = config.get('ams.flowKey') as string;
+    const axiosClient = axios.create(); // TODO: move AMS playback logic to API and use instead of flow above
+
+    delete axiosClient.defaults.headers.common['Ocp-Apim-Subscription-Key'];
+    delete axiosClient.defaults.headers.common['X-User-Id'];
```

Comment:
you may not want to do this. Does it wipe the headers for subsequent requests. For example, if you go back to `/browse` does it still work or does it no longer send the key and user id?
If you don't want to send these headers you can add them as null do the headers you are explicitly passing on line 59?

Reply:
ah i missed that 👍